utah-design-system-website/src/react/components/websiteContent/Search.jsx

```diff
@@ -19,9 +20,56 @@ export function Search() {
     if (param) { setQuery(param); }
   };
 
+  const checkSkipLink = () => {
+    const link = document.getElementById('skip-link');
+    if (link) {
+      setSkipLink(link);
+    } else {
+      // Create observer to watch for new nodes
+      const obs = new MutationObserver(() => {
+        const link = document.getElementById('skip-link');
```

Comment:
the `link` variable is declared twice. once on line 24 and once on line 30. while javascript allows this with scoping, it can be confusing for a developer reading it.